AppDB/zkappscale/zktransaction.py

```diff
@@ -1340,18 +1345,20 @@ def get_datastore_groomer_lock(self):
       
     return True
 
-  def release_datastore_groomer_lock(self):
-    """ Releases the datastore groomer lock. 
+  def release_lock_with_path(self, path):
+    """ Releases lock based on path.
    
+    Args:
+      path: A str, the lock path.
     Returns:
       True on success, False on system failures.
```

Comment:
This function says it returns True/False but in the function there are None returns.